docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -74,12 +74,12 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
     <tr>
       <td>Webhooks</td>
       <td>Read & Write</td>
-      <td>Codacy creates webhooks for organization and repository events (creation, deletion, member added, etc.) to enable <a href="../../organizations/what-are-synced-organizations/">synced organizations</a>.</td>
+      <td>Codacy creates webhooks for organization and repository events (creation, deletion, member added, etc.) to enable <a href="../../organizations/what-are-organizations/">organizations</a>.</td>
```

Comment:
Having the word organization repeated in the same sentence can be confusing. 

```suggestion
      <td>Codacy creates webhooks for organization and repository events (creation, deletion, member added, etc.) to enable Codacy <a href="../../organizations/what-are-organizations/">organizations</a>.</td>
```

Reply:
@prcr Can you please validate if my understanding is correct?

Reply:
I'm thinking that mentioning the part "to enable organizations" is now redundant because the product (as a whole) needs those permissions.

This is different from before, where you would only need those permissions if you were using the "synced" organizations, but you could still use the legacy "manual" organizations.

Reply:
Ohhh... That's completely different. Should it be like this?
 
```suggestion
      <td>Codacy creates webhooks for organization and repository events (creation, deletion, member added, etc.). </td>
```

Reply:
Yes, that's exactly what I meant. Things are simpler now that we only have one kind of organizations. :slightly_smiling_face: